### ISSUE TITLE
The formula for score, one step score,  

### ISSUE THREAD

**Issue by realRuiGuo** [arx-deidentifier/arx#269]
I want to know the formula for score:
`final String infoloss = node.getLowestScore().toString() +
                                        " [" + SWTUtil.getPrettyString(asRelativeValue(node.getLowestScore(), result)) + "%]"; //$NON-NLS-1$ //$NON-NLS-2$
                Property score = new Property(Resources.getMessage("PropertiesView.46"), new String[] { infoloss }); //$NON-NLS-1$`

**Which of the following classes is used to calculate the score above, that is, the score of the output property interface. It would be better if there is a formula**

![image](https://user-images.githubusercontent.com/12027656/57790618-b3037680-776d-11e9-95ac-240dfce46940.png)
![image](https://user-images.githubusercontent.com/12027656/57790831-1c838500-776e-11e9-9073-88d229d82573.png)

**is it right?:**
![image](https://user-images.githubusercontent.com/12027656/57793071-0e843300-7773-11e9-9b84-b1ffa3689f27.png)

![image](https://user-images.githubusercontent.com/12027656/57793128-2d82c500-7773-11e9-90d5-467bce292b64.png)

![image](https://user-images.githubusercontent.com/12027656/57793151-3a9fb400-7773-11e9-9946-edd756730033.png)

**Comment by prasser** [comment]
The formula depends on the metric for information loss that has been used to optimize output data (also called utility or quality model). The formula that you mentioned is the one used by the model MetricMDNMLoss.

**Comment by prasser** [comment]
Seems to be answered. Closing.
